### PR TITLE
[types/cesium/index.d.ts] added some missing types in Cryptographic types definition

### PR DESCRIPTION
**Pull request by UsamaMehmood**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated.

### CODE REVIEW

File: types/cesium/index.d.ts [hunk 1]
```diff
@@ -20,6 +21,9 @@ declare namespace Cesium {
         getURL(resource: string): string;
     }
 
+    let Ion: {defaultAccessToken: string, defaultServer: string};
+    let BingMapsApi: {defaultKey: string};
```
Comment (szechyjs) [inline types/cesium/index.d.ts]: these are not type definitions.

**Review by szechyjs — CHANGES_REQUESTED** [review]

**Comment by Crocsx** [comment]
@UsamaMehmood can you check the merge conflict ?

**Comment by UsamaMehmood** [comment]
Hi @Crocsx, I have fixed the merge conflict, you can proceed. Thanks.

**Review by Crocsx — APPROVED** [review]

**Comment by Crocsx** [comment]
@UsamaMehmood I do not have the merge right as it is automated. Some owner merge got executed before your. There are still have conflict to fix.

**Comment by Crocsx** [comment]
BTW, I just noted that I added this in another PR

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42113/files

so I guess this can be closed

**Review by Crocsx — CHANGES_REQUESTED** [review]
can be closed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42113

**Pull request closed without merging**